Formula/httpd.rb

```diff
@@ -0,0 +1,132 @@
+class Httpd < Formula
```

Comment:
Maybe `apache-httpd`, similar to other apache projects? When it was in homebrew/apache/httpd is was clearer, but now `httpd` seems a bit terse (and not the name it is usually referred to as).

Reply:
`httpd` is technically the name of the project and does seem to be the top hit when you search. Don't have strong feelings either way, though. @ilovezfs, any thoughts?

Reply:
Fedora/RedHat call it `httpd` and Debian calls it `apache2`, so some aliases might be in order here.